### PR TITLE
chore: script to run CI locally

### PR DESCRIPTION
**Pull request by doitian**

### CODE REVIEW

File: devtools/ci/local.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    body="${body}CI: success ✅\\n"
+  else
+    body="${body}CI: failure ❌\\n"
+  fi
+  if [ $integration_pass = true ]; then
+    body="${body}integration: success ✅\\n"
+  elif [ "$integration_pass" = false ]; then
+    body="${body}integration: failure ❌\\n"
+  fi
+  github_api "/repos/nervosnetwork/ckb/issues/$pr/comments" -X POST -d'{"body":"'"$body"'"}'
```
Comment (keroro520) [inline devtools/ci/local.sh]: `"/repos/nervosnetwork/ckb/issues/$pr/comments"`

or 

`"/repos/nervosnetwork/ckb/pulls/$pr/comments"`

?

**Comment by doitian** [comment]
@nervos-bot ci-status 1961de2a685dfc660b8a79b58bc606d49f380b56

CI: failure ❌

**Comment by doitian** [comment]
@nervos-bot give me five

**Comment by doitian** [comment]
@nervos-bot ci-status 1961de2a685dfc660b8a79b58bc606d49f380b56

CI: success ✅

File: devtools/ci/local.sh (hunk 1 above)
Comment (doitian) [reply devtools/ci/local.sh]: issues

**Comment by doitian** [comment]
@nervos-bot ci-status 1b613de39fff7f46296380c787f351c003168563

CI: success ✅

**Pull request merged**